ppx/html.ml

```diff
@@ -15,558 +17,1859 @@ type eventType =
+  ; { tag= "colgroup"
+    ; attributes= domAttributes & attributesHTML & colgroupHTMLAttributes }
+  ; { tag= "data"
+    ; attributes= domAttributes & attributesHTML & dataHTMLAttributes }
+  ; {tag= "datalist"; attributes= domAttributes & attributesHTML}
+  ; {tag= "dd"; attributes= domAttributes & attributesHTML}
+  ; {tag= "del"; attributes= domAttributes & attributesHTML & delHTMLAttributes}
+  ; { tag= "details"
+    ; attributes= domAttributes & attributesHTML & detailsHTMLAttributes }
+  ; {tag= "dfn"; attributes= domAttributes & attributesHTML}
+  ; { tag= "dialog"
+    ; attributes= domAttributes & attributesHTML & dialogHTMLAttributes }
+  ; {tag= "div"; attributes= domAttributes & attributesHTML}
+  ; {tag= "dl"; attributes= domAttributes & attributesHTML}
+  ; {tag= "dt"; attributes= domAttributes & attributesHTML}
+  ; {tag= "em"; attributes= domAttributes & attributesHTML}
+  ; { tag= "embed"
+    ; attributes= domAttributes & attributesHTML & embedHTMLAttributes }
+  ; { tag= "fieldset"
+    ; attributes= domAttributes & attributesHTML & fieldsetHTMLAttributes }
+  ; {tag= "figcaption"; attributes= domAttributes & attributesHTML}
+  ; {tag= "figure"; attributes= domAttributes & attributesHTML}
+  ; {tag= "footer"; attributes= domAttributes & attributesHTML}
+  ; { tag= "form"
+    ; attributes= domAttributes & attributesHTML & formHTMLAttributes }
+  ; {tag= "h1"; attributes= domAttributes & attributesHTML}
+  ; {tag= "h2"; attributes= domAttributes & attributesHTML}
+  ; {tag= "h3"; attributes= domAttributes & attributesHTML}
+  ; {tag= "h4"; attributes= domAttributes & attributesHTML}
+  ; {tag= "h5"; attributes= domAttributes & attributesHTML}
+  ; {tag= "h6"; attributes= domAttributes & attributesHTML}
+  ; {tag= "head"; attributes= domAttributes & attributesHTML}
+  ; {tag= "header"; attributes= domAttributes & attributesHTML}
+  ; {tag= "hgroup"; attributes= domAttributes & attributesHTML}
+  ; {tag= "hr"; attributes= domAttributes & attributesHTML}
+  ; { tag= "html"
+    ; attributes= domAttributes & attributesHTML & htmlHTMLAttributes }
+  ; {tag= "i"; attributes= domAttributes & attributesHTML}
+  ; { tag= "iframe"
+    ; attributes= domAttributes & attributesHTML & iframeHTMLAttributes }
+  ; {tag= "img"; attributes= domAttributes & attributesHTML & imgHTMLAttributes}
+  ; { tag= "input"
+    ; attributes= domAttributes & attributesHTML & inputHTMLAttributes }
+  ; {tag= "ins"; attributes= domAttributes & attributesHTML & insHTMLAttributes}
+  ; {tag= "kbd"; attributes= domAttributes & attributesHTML & domAttributes}
+  ; { tag= "keygen"
+    ; attributes= domAttributes & attributesHTML & keygenHTMLAttributes }
+  ; { tag= "label"
+    ; attributes= domAttributes & attributesHTML & labelHTMLAttributes }
+  ; {tag= "legend"; attributes= domAttributes & attributesHTML}
+  ; {tag= "li"; attributes= domAttributes & attributesHTML & liHTMLAttributes}
```

Comment:
Does it make sense to just inline `liHTMLAttributes` here? as they're only used once...

Reply:
Most attributes are moved directly from TypeScript. One of the lasts tasks I want to do is `[ ] Refactor names of the groups of attributes (domAttributes, reactAttributes, attributesHTML) Try to came up with the HTML Spec terms (add links to it)`